### DISCUSSION TITLE
Road map in relation to Owncloud OICS

### DISCUSSION THREAD

**Discussion by lil5** [schoentoon/go-cloud#11] [General]
https://github.com/owncloud/ocis

Have you heard of this project?

How does go cloud's goals differ from Owncloud's

**Comment by schoentoon** [comment]
I have not heard of this project, I will look into it and comment back on it later

**Reply by schoentoon** [reply]
So I had a brief look. Also tried to run it locally, which sadly seemed broken (but I won't judge them on that, as it is a tech preview). I think one of the bigger things that is a part of their target is federated storage. Which can certainly have it's uses, but I personally don't think it's all that needed in a storage system. Especially considering that I am mostly writing go-cloud for own use (where I would run it on a single server, for illustration I'm currently running Nextcloud personally for a total of 3 people). (Don't get me wrong, federation as a whole is great. I just don't think it should be tucked onto everything). And I kind of think there is the biggest difference. My aim being a solid system you would maybe use for your family/friends, which should be easy to maintain and keep up to date. Where as especially owncloud ocis that you linked seems to more aim towards big companies/schools.

I would however like to stress that I started on this like 2 months ago and it is still very much in alpha stage, so things are very likely to break still.

**Reply by lil5** [reply]
I think you'd be surprised how many people have the same requirements.

Great to hear your thoughts on it
